### ISSUE TITLE
Support for Windows scoop.sh installer

### ISSUE THREAD

**Issue by JaneX8** [learningequality/kolibri#12834]
I'd like to be able to do `scoop install kolibri` using the [scoop.sh installer for Windows](https://scoop.sh/).

**Comment by MisRob** [comment]
@jredrejo @rtibbles Any thoughts?

**Comment by jredrejo** [comment]
No, never heard of this installer. It could be a community project if there's interest on it and someone has good Powershell skills

**Comment by rtibbles** [comment]
The main issue is that scoop.sh (from its documentation) focuses on portable/standalone apps. Currently on Windows, Kolibri has an explicit dependency on Python, so we would need to create a working PyInstaller build of Kolibri for Windows before this is possible.

**Comment by JaneX8** [comment]
Or use a `venv`, similar to this unofficial scoop repo dealing with Python: https://github.com/jcwillox/scoop-python/blob/master/bucket/black.json.

**Comment by rtibbles** [comment]
OK, this seems like quite a stretch as an installer, with a low potential reach. Happy to get PRs for it, but I doubt we will prioritize this internally for the time being.
